templates/availability/_table.html.twig

```diff
@@ -48,7 +48,7 @@
         <tbody>
         {% for i in 0..nbChoicesPerDay-1 %}
             <tr>
-                <td class="hour-title">{{ choices[i].date|date('H') }}-{{ choices[i+1].date|date('H') }}h</td>
+                <td class="hour-title">{{ choices[i].date|date('H:i') }} - {{ choices[i+1].date|date('H:i') }}</td>
```

Comment:
this text is too large for mobile phones. We should maybe display the minutes only if needed